### PR TITLE
removed jewel and compose shadowing

### PR DESCRIPTION
**Pull request by fscarponi**
removed shadowing for "jewel" and "compose"
set compose import to compileOnly, because the platform does not contain androidx.ui.tooling package

### CODE REVIEW

File: plugin/build.gradle.kts [hunk 1]
```diff
@@ -49,13 +49,12 @@ val tooling: Configuration by configurations.creating {
 }
 
 dependencies {
-    implementation(compose.desktop.linux_arm64)
-    implementation(compose.desktop.linux_x64)
-    implementation(compose.desktop.macos_arm64)
-    implementation(compose.desktop.macos_x64)
-    implementation(compose.desktop.windows_x64)
+    compileOnly(compose.desktop.linux_arm64)
```
Comment (lamba92) [inline plugin/build.gradle.kts:52]: I believe that we can remove the runtimes altogether. Isn't the preview artifact available separately?

File: plugin/src/main/kotlin/com/jetbrains/packagesearch/plugin/ui/panels/tree/PackageSearchModulesTree.kt [hunk 2]
```diff
@@ -58,7 +58,7 @@ fun PackageSearchModulesTree(
         onExpandAll = viewModel::expandAll,
         onCollapseAll = {
             val rootIds = tree.roots.map { it.id }.toSet()
-            viewModel.treeState.selectedKeys = viewModel.treeState.selectedKeys intersect rootIds
+            viewModel.treeState.selectedKeys =( viewModel.treeState.selectedKeys intersect rootIds).toList()
```
Comment (lamba92) [inline plugin/src/main/kotlin/com/jetbrains/packagesearch/plugin/ui/panels/tree/PackageSearchModulesTree.kt:61]: Given that the keys are using lists, I assume that this is a quite old version...

**Review by lamba92 — CHANGES_REQUESTED** [review]

**Comment by lamba92** [comment]
[Uhmmm.....](https://stackoverflow.com/questions/52200096/github-pull-request-waiting-for-status-to-be-reported)
```
Some checks haven’t completed yet
1 expected check
build Expected — Waiting for status to be reported
```

File: plugin/src/main/kotlin/com/jetbrains/packagesearch/plugin/ui/panels/tree/PackageSearchModulesTree.kt (hunk 2 above)
Comment (fscarponi) [reply plugin/src/main/kotlin/com/jetbrains/packagesearch/plugin/ui/panels/tree/PackageSearchModulesTree.kt:61]: is in jewel foundation so nothing to do… we need to wait update to have the Set

File: plugin/build.gradle.kts (hunk 1 above)
Comment (fscarponi) [reply plugin/build.gradle.kts]: found here https://mvnrepository.com/repos/google
[lib](https://mvnrepository.com/artifact/androidx.compose.ui/ui-tooling-preview) but not sure if need something else, i'm not able to import it correctly

solved by loading compose.desktop.currentOS

**Pull request merged**